templates/advancement/class-advancement-summary.hbs

```diff
@@ -4,7 +4,8 @@
       {{#if learning}}
         {{localize "ED.Dialogs.Legend.ClassAdvancement.learningClass" type=classItem.type name=classItem.name}}
       {{else}}
-        {{localize "ED.Dialogs.Legend.ClassAdvancement.increaseClassTo" nextLevel}}
+        {{localize "ED.Dialogs.Legend.ClassAdvancement.increaseClassTo"}}{{nextLevel}}
```

Comment:
should be a `format` if it depends on input. in this case, handlebars, probably something like `{{localize "ED.Dialogs.Legend.ClassAdvancement.increaseClassTo" nextLevel=nextLevel}}`

Reply:
your example is not working anything inside of the localize handlebar does not do anything... what is the problem to add the {{nextLevel}}?

Reply:
ok, got it to work, great!